index.js

```diff
@@ -10,15 +10,29 @@ const fileRead = async (fileName) => {
   return fileData;
 };
 
+// Function to return the words of the data provided.
+const fileWords = async (data) => {
+  const words = data.split(" ").length;
+  console.log(`${chalk.yellowBright.bold("Number of words: " + words)}`);
+};
+
+// Function to return the sentences of the data provided.
+const fileSentences = async (data) => {
+  const sentences = data.match(/[\w|\)][.?!](\s|$)/g).length;
```

Comment:
change coloring to match "-h" command style style

Reply:
is this acceptable?
![image](https://user-images.githubusercontent.com/54966170/196299463-5f408858-c260-4de9-9030-d3968d371596.png)

Reply:
sure!